helm/idol-nifi/templates/nifi/nifi.yaml

```diff
@@ -8,37 +8,19 @@
 # The information contained herein is subject to change without notice.
 #
 # END COPYRIGHT NOTICE
-apiVersion: apps/v1
-kind: StatefulSet
-metadata:
-  name: idol-nifi
-  labels: {{- include "idol-library.labels" (dict "root" . "component" .Values.nifi) | nindent 4 -}}
-    name: nifi
-    app: nifi
```

Comment:
I think you need metadata.labels.app, else various selectors will fail (e.g. the nifi service created by headless-service.yaml)

Reply:
Though looking at the headless-service.yaml may also need an explicit override to set `spec.serviceName: nifi` (rather than it being based on $component.name)

Reply:
Making these changes seems to break ingress, as I now get 503 errors when accessing localhost:12443/nifi/